docs/csharp/properties.md

```diff
@@ -153,6 +177,35 @@ can also be declared `protected`, `internal`, `protected internal` or even `priv
 It is also legal to place the more restrictive modifier on the `get` accessor. For example, you could
 have a `public` property, but restrict the `get` accessor to `private`. That scenario is rarely done
 in practice.
+
+You can also restrict modifications to a property so that it can only be set in a constructor
+or a field initializer. You can modify the `Person` class so as follows:
```

Comment:
Did you mean "property initializer"? I don't see how could you modify get-only auto-property from a field initializer.